pkgs/os-specific/bsd/freebsd/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, stdenvNoCC
+, pkgsBuildBuild, pkgsBuildHost, pkgsBuildTarget, pkgsHostHost, pkgsHostTarget, pkgsTargetTarget
+, splicePackages, buildPackages
+, fetchsvn, lib, groff, mandoc, zlib, yacc, flex
+, writeText, symlinkJoin
+}:
+
+let
+  version = "12";
+
+  fetchFreeBSD = path: sha256: fetchsvn {
+    url = "https://svn.freebsd.org/base/stable/${version}/${path}";
```

Comment:
`stable` branch [may not be](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/current-stable.html) stable. You can use [one of the releng branch](https://svnweb.freebsd.org/base/releng/12.1/) to track released code. Also you might want to stick to a particular svn revision, and not HEAD

Reply:
Thanks!